### PR TITLE
Grouping update for demo

### PR DESCRIPTION
**Pull request by dean-microsoft**

### CODE REVIEW

File: scenariogroups/ExampleGroup.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
     "title": "Sample Solution Group",
```
Comment (unrealab) [inline scenariogroups/ExampleGroup.json:2]: DO you want to change the title ?

File: scenarios/microsoft-azure-arcbox-v1/discovery.metadata.json [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
-  "shortDescription": "This sample scenario template helps illustrate how the metadata will show up in solutions center",
+  "shortDescription": "Use ArcBox to deploy an Azure Arc sandbox in less than an hour.",
   "cost": null,
-  "activationString": "Click to view details of the Solution",
+  "activationString": "View details of the solution",
   "keywords": ["Sample","solution", "template"],
-  "detailsDescription": "Soe more contextual details about what happens with this solution and why this is useful and why a user should use it.",
-  "learnMoreLink": "aka.ms/learnmoreaboutme",
-  "learnMoreLinkText": "Learn more about Sample solution",
+  "detailsDescription": "See more contextual details about what happens with this solution and why this is useful and why a user should use it.",
```
Comment (unrealab) [inline scenarios/microsoft-azure-arcbox-v1/discovery.metadata.json:8]: "...and why you would use it " perhaps ? You are directly addressing the user here.

**Review by unrealab — CHANGES_REQUESTED** [review]

**Comment by unrealab** [comment]
Please raise against staging.

File: scenarios/microsoft-azure-arcbox-v1/discovery.metadata.json (hunk 2 above)
Comment (dean-microsoft) [reply scenarios/microsoft-azure-arcbox-v1/discovery.metadata.json:8]: i don't know where this is displayed, i just left the default and fixed "Soe" to "See"

File: scenariogroups/ExampleGroup.json (hunk 1 above)
Comment (dean-microsoft) [reply scenariogroups/ExampleGroup.json:2]: Nope!

**Review by unrealab — APPROVED** [review]

**Pull request merged**